core/commands/object.go

```diff
@@ -344,39 +344,21 @@ Data should be in the format specified by the --inputenc flag.
 }
 
 // objectData takes a key string and writes out the raw bytes of that node (if there is one)
-func objectData(n *core.IpfsNode, fpath path.Path) (io.Reader, error) {
-	dagnode, err := core.Resolve(n, fpath)
+func objectData(n *dag.Node, err error) (io.Reader, error) {
```

Comment:
Thought so too, this is mixed up from PR#1189 though. I'll remove it there and rebase this once it is merged.